static/tabs/instructions.html

```diff
@@ -0,0 +1,48 @@
+<div class="tab-pane fade" id="instructions" role="tabpanel" aria-labelledby="instructions-tab">
+    <div class="container mb-4">
+        <div class="row g-3">
+            <div class="col-lg-6">
+                <h2>Connecting PrusaSlicer/SuperSlicer</h2>
```

Comment:
Semantically speaking this should be an h1. We can apply h2 styling via boostrap

Reply:
Oh, good catch. I'll fix!

Reply:
Done :white_check_mark: